libpod/runtime_volume_linux.go

```diff
@@ -106,6 +107,26 @@ func (r *Runtime) newVolume(ctx context.Context, options ...VolumeCreateOption)
 		if err := LabelVolumePath(fullVolPath); err != nil {
 			return nil, err
 		}
+		projectQuotaSupported := false
+
+		q, err := quota.NewControl(r.config.Engine.VolumePath)
+		if err == nil {
+			projectQuotaSupported = true
+		}
+		quota := quota.Quota{}
+		if volume.config.Size > 0 || volume.config.Inodes > 0 {
+			if !projectQuotaSupported {
+				return nil, errors.New("Volume options size and inodes not supported. Filesystem does not support Project Quota")
+			}
+			quota.Size = volume.config.Size
+			quota.Inodes = volume.config.Inodes
+		}
+		if projectQuotaSupported {
```

Comment:
```suggestion
```
Just like this? We don’t need to set quota (and potentially conflict with XFS project IDs).

Reply:
don’t need to set quota … if there isn’t a quota request.

Reply:
… actually, all of this, including the quota support check, can be wrapped inside the `volume.config.{Size,Inodes} > 0` condition.

/me steps away to regain focus.